src/index.js

```diff
@@ -256,24 +271,28 @@ export default {
           command.push(filePath)
         }
 
-        const output = await executeRubocop(execOptions, command)
-
-        // Process was canceled by newer process
-        if (output === null) { return null }
+        let output
+        try {
+          output = await executeRubocop(execOptions, command)
```

Comment:
I think this is incorrect because all errors will be shown onto the linter pane.